packages/expo-updates/src/Updates.types.ts

```diff
@@ -97,10 +97,23 @@ type UpdateFetchResultFailure = {
   manifest: undefined;
 };
 
+/**
+ * The rollback to embedded result of fetching a new update.
+ */
+type UpdateFetchResultRollbackToEmbedded = {
+  /**
+   * Signifies that the update was a roll back to the embedded update.
```

Comment:
```suggestion
   * Signifies that the update was a rollback to the embedded update.
```

Reply:
We decided somewhere previously that "roll" and "back" should be different words (`RollBackToEmbeddedUpdateDirective`).